src/components/user/MenuLeft.js

```diff
@@ -34,11 +37,16 @@ export default class MenuLeft extends PureComponent {
             </Link>
           </div>
           <div className="content create-target">
-            {section === newTarget ? (
-              <CreateTargetForm onSubmit={handleCreateTarget} topics={topicList} />
-            ) : (
+            {section === newTarget ?
+              <CreateTargetForm
+                onSubmit={isDeletingTarget ? handleDeleteTarget : handleCreateTarget}
+                topics={topicList}
+                isDeletingTarget={isDeletingTarget}
+                enableReinitialize={enableReinitialize}
```

Comment:
If enableReinitialize is always true you could just put `<CreateTargetForm enableReinitialize />`, you don't need to define and assign the variable in line 24. Unless you think it may change in the future.